BankChain/app/src/androidTest/java/nl/tudelft/ewi/ds/bankchain/bank/bunq/BunqTest.java

```diff
@@ -144,4 +153,100 @@ public void testExceptionConfirmationNotFuture() {
 
         assertTrue(bank.confirmException(thr) == thr);
     }
+
+    @Test
+    public void testGetUsers() throws ExecutionException, InterruptedException {
```

Comment:
Please also add tests for bad code paths and any exceptions that might be thrown.